hdk-rust/src/panic_handler.rs

```diff
@@ -0,0 +1,24 @@
+use crate::api::debug;
+use holochain_core_types::json::RawString;
+use std::panic;
+
+#[no_mangle]
+pub extern "C" fn __install_panic_handler() -> () {
+    panic::set_hook(Box::new(move |info| {
+        let _ = debug(RawString::from(
+            info.payload().downcast_ref::<String>().unwrap().clone(),
+        ));
+        //let _ = debug(RawString::from(format!("{}", info.message().unwrap().clone())));
+        let _ = if let Some(location) = info.location() {
+            debug(RawString::from(format!(
+                "panic occurred in file '{}' at line {}",
+                location.file(),
+                location.line()
+            )))
+        } else {
+            debug(RawString::from(format!(
+                "panic occurred but can't get location information..."
```

Comment:
If we know this is going to happen every time, we should just comment this out?

Reply:
It's not going to happen every time. With debug symbols we actually see the file and line number. See above output that I got.